chunky/src/java/se/llbit/chunky/world/region/RegionParser.java

```diff
@@ -53,6 +57,11 @@ public RegionParser(WorldMapLoader loader, RegionQueue queue, MapView mapView) {
   }
 
   @Override public void run() {
+    //setting load factor to decrease expensive hash collisions between BlockSpec objects
```

Comment:
Maybe we could take a look at [jo-nbt](https://github.com/llbit/jo-nbt) to see if there are any possible optimizations? The `equals` and `hashCode` methods there look good to me, though.